### ISSUE TITLE
Clipboard from local to remote buggy

### ISSUE THREAD

**Issue by romanad** [cedrozor/myrtille#158]
Hello dear support team,
The app is amazing, very useful and nice.
But there is some little problem with clipboard from local to remote - if I trying to send some text from local (for example first paragraph from http://users.ox.ac.uk/~eets/) there are several bugs in pasted text. 
Does it happens only with me or it's common issue?

**Comment by cedrozor** [comment]
It can happen if myrtille is running in HTML4 compatibility mode (which is the fallback if you didn't enabled websockets into IIS) in case of large blocks of text, because HTML4 use XmlHttp requests and these are limited in size (unlike websockets, which can handle much more data).

To enable websockets into IIS (they are not enabled by default), follow this link: https://www.iis.net/configreference/system.webserver/websocket

It may also be due to something else. I fixed and improved a lot of things into the latest version (2.5.5).

**Comment by romanad** [comment]
OK, thank you for rapid response, and new version. I already installed it.
It seems I have an issue with websocket protocol processing at HTML 4 and HTML 5 (I use HTTPS connection for security reason). 
It looks like that websocket enables only when I using HTML 4, and when I use HTML 5 - I  see LONGPOLLING in stats.
Maybe I just doing something wrong?
Please see screenshots attached.
![2 5 5](https://user-images.githubusercontent.com/26842711/58785850-0a6b6880-85ef-11e9-9786-3fe0f854ade2.jpg)
![Windows Features - websocket](https://user-images.githubusercontent.com/26842711/58785856-0e978600-85ef-11e9-95aa-aa86ff055386.jpg)
![HTML_4 websocket](https://user-images.githubusercontent.com/26842711/58785860-10f9e000-85ef-11e9-9a1b-8be968106885.jpg)
![HTML_5_Longpolling](https://user-images.githubusercontent.com/26842711/58785866-12c3a380-85ef-11e9-84b6-2f8a1b5f6327.jpg)

**Comment by cedrozor** [comment]
This issue is related to https://github.com/cedrozor/myrtille/issues/159 and https://github.com/cedrozor/myrtille/issues/160

I will have base64 encoding both ways and that's all :)

**Comment by cedrozor** [comment]
The next version will have a seamless clipboard synchronization based on the async clipboard API (Chrome + HTTPS only for now) with a fallback to the standard web API (document.execCommand, with user action) for other browsers. It uses unicode characters, so should work whatever the charset (I will ensure that before release).

I will also have a look to the unicode translation of the javascript keycodes for keyboard inputs, because there are some issues with the Chinese charset (https://github.com/cedrozor/myrtille/issues/170). The "keyboard" popup uses the same routines, so is also impacted.

**Comment by romanad** [comment]
Thank you very much Sir!
I'm very anticipating for the new version of Myrtille. 
That issue is have to be resolved.

**Comment by cedrozor** [comment]
Fixed in the upcoming version (2.6), with full UTF-8 (unicode) support (Latin, Cyrillic, Chinese, etc.). Clipboards (local and remote) are now automatically synchronized.

I also changed the "Clipboard" popup to send some text into the remote clipboard; this will avoid using the "Keyboard" popup which send some text quite slowly, because character per character, into the remote session (current application).
